File: backend/GeoQuiz_backend/GeoQuiz_backend/Application/DTOs/KingOfTheHill/PlayerEliminatedData.cs
namespace GeoQuiz_backend.Application.DTOs.KingOfTheHill
{
    public class PlayerEliminatedData
    {
        public Guid PlayerId { get; set; }
        public int RoundsSurvived { get; set; }
        public int Place { get; set; } 
        public int CorrectAnswers { get; set; }
        public int TotalScore { get; set; }
    }
}

File: frontend/app/src/main/java/com/example/geoquiz_frontend/data/remote/dtos/koth/PlayerEliminatedData.java
package com.example.geoquiz_frontend.data.remote.dtos.koth;

public class PlayerEliminatedData {
    private String playerId;
    private int roundsSurvived;
    private int place;
    private int correctAnswers;
    private int totalScore;

    public String getPlayerId() { return playerId; }
    public int getRoundsSurvived() { return roundsSurvived; }
    public int getPlace() { return place; }
    public int getCorrectAnswers() { return correctAnswers; }
    public int getTotalScore() { return totalScore; }
}
